### PR TITLE
Add osaka0 overlays

### PR DESCRIPTION
**Pull request by dulltz**
https://github.com/cybozu-go/neco/issues/519

### CODE REVIEW

File: metallb/overlays/osaka0/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      - 10.78.32.0/20
+    - name: bastion
+      protocol: bgp
+      addresses:
+      - 10.78.48.240/28
+      auto-assign: false
+    - name: internet
+      protocol: bgp
+      addresses:
+      - 202.253.115.208/28
```
Comment (masa213f) [inline metallb/overlays/osaka0/configmap.yaml]: Why don't you use `202.253.115.192/28`?
Is there any reason?

Comment (dulltz) [reply metallb/overlays/osaka0/configmap.yaml]: We will use `202.253.115.192/28` for coil.

Comment (dulltz) [reply metallb/overlays/osaka0/configmap.yaml]: We will use `202.253.115.192/28` for Coil.

Comment (masa213f) [reply metallb/overlays/osaka0/configmap.yaml]: OK. Thank you!

**Pull request merged**